### ISSUE TITLE
React import now required for stories using JSX decorators

### ISSUE THREAD

**Issue by kmonahan** [forumone/gesso#831]
Something with the Storybook 8 upgrade has caused stories that use JSX in a decorator (example: button.stories.jsx) to individually import React in that file. Error message that displays otherwise seems to be connected to `pmmmwh/react-refresh-webpack-plugin`. Desired behavior is to not have to keep importing React to use JSX.

**Comment by coreylafferty** [comment]
To temporarily remedy this, I added `import React from 'react';` to the top of the following stories:

```
back-to-top.stories.jsx
button.stories.jsx
icon.stories.jsx
```

Additionally, in gesso-uswds, I also added it to the following stories in the USWDS folder:
```
alert.stories.jsx
button.stories.jsx
```

**Comment by dcmouyard** [comment]
PR I just created might fix this issue: #837
